TeachingRecordSystem/tests/TeachingRecordSystem.SupportUi.Tests/HostFixture.cs

```diff
@@ -76,6 +79,13 @@ protected override void ConfigureWebHost(IWebHostBuilder builder)
             services.AddTransient<ICurrentUserIdProvider, TestUserCurrentUserIdProvider>();
             services.AddSingleton<FakeTrnGenerator>();
             services.AddSingleton<TrsDataSyncHelper>();
+            FileService
```

Comment:
As above, can we bundle this mock creation into a local function?

Reply:
Done